cost/aws/unused_rds/unused_rds.pt

```diff
@@ -509,16 +529,21 @@ script "js_instance_cost_mapping", type:"javascript" do
       }
       var total_savings=0;
       _.each(instance_list, function(instance){
+        console.log(instance)
```

Comment:
Not sure if we want to remove this `console.log` line now?

Reply:
thanks jim! good catch.